cmd/cli/uninstall_mesh.go

```diff
@@ -119,12 +119,9 @@ func (d *uninstallMeshCmd) run() error {
 			return nil
 		}
 		// Searches for the mesh specified by the mesh-name flag if specified
-		specifiedMeshFound := false
-		if d.meshName != "" {
-			specifiedMeshFound = d.findMesh(meshInfoList)
-			if !specifiedMeshFound {
-				return nil
-			}
+		meshFlagSpecified, specifiedMeshFound := d.findFlagSpecifiedMesh(meshInfoList)
+		if meshFlagSpecified && !specifiedMeshFound {
```

Comment:
if these values are just logically compared, we should be able to return a single boolean from the method findFlagSpecifiedMesh